includes/definitions/discovery/lenovoXClarityIMPI.yaml

```diff
@@ -0,0 +1,66 @@
+mib: LENOVO-XCC-MIB:LENOVO-XCC-ALERT-MIB:LENOVO-SMI-MIB:LENOVO-PRODUCTS-MIB
+modules: 
+    os:
+        hardware: LENOVO-XCC-MIB::machineLevelProductName.0
+        serial: LENOVO-XCC-MIB::machineLevelSerialNumber.0
+        version: LENOVO-XCC-MIB::xccVpdVersionString.1    
+    sensors: 
+        temperature: 
+            data: 
+                - 
+                    oid: temperature
+                    value: tempReading
+                    #num_oid: '.1.3.6.1.4.1.19046.11.1.1.1.2.'
```

Comment:
you need to define `num_oid` for the tests to pass. 
Probably something like : 
`num_oid: '.1.3.6.1.4.1.19046.11.1.1.1.2.{{ $index }}' `

Reply:
(if you look at the debug data displayed during discovery (`./discovery.php -v -d -h <deviceID> -m sensors`), you should have hints on which num_oid was computed automatically out of the MIB file.

Reply:
Done! Thank you for your explanation!